Makefile

```diff
@@ -335,13 +335,13 @@ redeploy: images load-images ## Redeploy nexodus after images changes
 init-db:
 # wait for the DB to be up, then restart the services that use it.
 ifeq ($(OVERLAY),dev)
-# todo: find a reliable way to wait for the crunchy based deployment of postgres to be online
+	$(CMD_PREFIX) kubectl wait -n nexodus crdbcluster/cockroachdb --timeout=15m --for=condition=StanzaCreated || true
```

Comment:
you've got cockroachdb here in the normal dev overlay (postgres)

Reply:
doh